core/contextmenu.js

```diff
@@ -408,12 +408,14 @@ Blockly.ContextMenu.wsExpandOption = function(hasCollapsedBlocks, topBlocks) {
  * Make a context menu option for deleting the current workspace comment.
  * @param {!Blockly.WorkspaceCommentSvg} comment The workspace comment where the
  *     right-click originated.
+ * @param {?string} optLabel An optional label for the comment. Defaults to
+ * Blockly.Msg.REMOVE_COMMENT
```

Comment:
Indent line continuation +4 spaces.

What else would this text be?  

It's the text of the context menu option, not the label for the comment.

Reply:
Yeah, I had originally changed that so that block comment context menu should read 'Delete' instead of 'Remove Comment', but then I realized that workspace comments should get the same context menu option label. Fixed.